content/200-concepts/100-components/drivers.mdx

```diff
@@ -0,0 +1,134 @@
+---
+title: 'Drivers'
+metaTitle: 'Drivers (Preview)'
+metaDescription: 'Learn how you can use Prisma with other database drivers using driver adapters (Preview)'
+preview: true
+tocDepth: 3
+---
+
+## Built-in drivers (default)
+
+One of Prisma Client's component's is the [Query Engine](./prisma-engines/query-engine) <span class="concept"></span>. The Query Engine transforms Prisma Client queries to SQL statements that are executed using the included database driver. By default, the Query Engine uses TCP connections to run a query against your database.
+
+<!-- TODO: add image here -->
```

Comment:
HTML comments break the GitHub highlights for `.mdx` files:
<img width="1169" alt="image" src="https://github.com/prisma/docs/assets/183673/f56ebeba-12fd-43bb-adda-be69498668b9">